.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: ruby
+rvm:
+  - 2.4.1
+
+script:
+  - JEKYLL_ENV=production bundle exec jekyll build --destination site
+  
+sudo: false # route your build to the container-based infrastructure for a faster build
+
+cache: bundler # caching bundler gem packages will speed up build
+exclude: [vendor]
+
+install:
+- bundle install
+
+deploy:
```

Comment:
I don't think we want this section. What this will do is when someone pushes to the dev branch travis CI will then push a build of the site to the master branch.